config/default.schema.json

```diff
@@ -453,6 +453,23 @@
 			"description": "Synchronous encryption key used to secure search user passwords",
 			"pattern": ".{16}.*"
 		},
+		"AES_KEY": {
+			"type": "string",
+			"description": "Symetric encryption key used to encrypt and decrypt secrets.",
+			"pattern": ".{16}.*"
+		},
+		"SENTRY_DSN": {
```

Comment:
this seems to be a merge conflict, sentry has been removed on main.

Reply:
Yes it did. Now it also has been removed from the branch. thx for spotting.